tools/c7n_tencentcloud/c7n_tencentcloud/query.py

```diff
@@ -39,9 +39,17 @@ class ResourceTypeInfo(metaclass=TypeMeta):
     batch_size: int = 10
 
     # used by metric filter
+    metrics_enabled: bool = False
```

Comment:
isn't this automatically implied by having metrics_namespace value set?

Reply:
just want to use a explicit flag here